3d/pcb.scad

```diff
@@ -115,11 +115,22 @@ module pcb() {
 // 2D cutouts needed to mount the PCB module, origin at the center of the mounting hole
 module pcb_cutouts() {
     hull_slide() {
-        translate([-m4_hole_diameter/2, -m4_hole_diameter/2]) {
-            square([m4_hole_diameter/2 + pcb_hole_to_sensor_pin_1_x + sensor_pin_pitch, m4_hole_diameter]);
+        // Bolt slot
+        hull() {
+            circle(r=m4_hole_diameter/2, $fn=30);
+            translate([pcb_hole_to_sensor_pin_1_x + sensor_pin_pitch - m4_hole_diameter/2, 0, 0])
+                circle(r=m4_hole_diameter/2, $fn=30);
         }
+        // Pin header slot
         translate([pcb_hole_to_connector_pin_1_x - connector_pin_pitch, -pcb_hole_to_connector_pin_1_x]) {
```

Comment:
Unrelated to your change, but now that I'm reviewing this again it seems like this should be `-pcb_hole_to_connector_pin_1_y` instead of `_x` (doesn't practically matter since the x/y offset are currently both 8.636)

Reply:
Fixed in 488fb929b6e9dbd9271ee29171b7076ad83dd423